apps/nextjs-website/react-components/types/common/Common.types.ts

```diff
@@ -22,5 +22,5 @@ export const isJSX = <T>(arg: T | Generic): arg is Generic =>
 
 export interface CtaButtonProps extends Partial<ButtonProps> {
   readonly text: string;
-  variant?: 'contained' | 'outlined';
+  variant?: 'contained' | 'outlined' | 'text';
```

Comment:
```suggestion
  variant?: 'contained' | 'outlined' | 'naked';
```
Since we're styling PreHeader buttons to look like a 'naked' variant, can't we use the 'naked' variant directly?

Is there anything that stops us from being able to do so?

I know when PreHeader was inside EC we had to use the 'text' variant and style it accordingly, but I don't see that being a limitation anymore.